CHANGES.rst

```diff
@@ -311,6 +311,8 @@ astropy.visualization
 astropy.wcs
 ^^^^^^^^^^^
 
+- Fixed bug in WCS.wcs.cunit equality check [#8480]
```

Comment:
Could be reworded, because it actually enables equality testing based on the contents of the class.